.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,17 @@
+# Apply various labels on PRs
+
+name: pr-labeler
+on:
+  pull_request:
+    types: [ opened ]
+
+jobs:
+  auto-approve:
+    if: github.event.pull_request.user.login == 'dependabot[bot]' || github.event.pull_request.user.login == 'dependabot-preview[bot]'
```

Comment:
I believe it should be possible to configure dependabot to add that label on its pull requests so this workflow shouldn't be required

Reply:
Its not possible because security related PR's are not coupled with any configuration file, and controlled by the *Dependabot security updates* setting in the [security analysis](https://github.com/aws/aws-cdk/settings/security_analysis) page.

Note that dependabot right now is only explicitly configured to upgrade github actions:

https://github.com/aws/aws-cdk/blob/e26f5befc2adedeb524fd263424c7920989b2288/.github/dependabot.yml#L5-L13

It does however still create security upgrade PR's: https://github.com/aws/aws-cdk/pull/16421

We decided to keep creating these PR's as a fallback to when our own dependency upgrade PR's is misbehaving. 
Unfortunately those security related PR's are not configurable, from what I could gather.